ui/src/app/pages/admin/admin-workspace-audit.tsx

```diff
@@ -31,7 +31,7 @@ const getAdminPageUrl = (subject: string) => {
 };
 
 export const WorkspaceAudit = () => {
-  const {workspaceNamespace = ''} = useParams();
+  const {workspaceNamespace = ''} = useParams<{ workspaceNamespace: string}>();
```

Comment:
nit: would remove leading space here: `{ workspaceNamespace:...`, I think most of the code does not follow that pattern. Here and throughout the PR.

Reply:
Done